pymeasure/instruments/instrument.py

```diff
@@ -149,38 +273,88 @@ def binary_values(self, command, header_bytes=0, dtype=np.float32):
         return self.adapter.binary_values(command, header_bytes, dtype)
 
     @staticmethod
-    def control(get_command, set_command, docs,
-                validator=lambda v, vs: v, values=(), map_values=False,
-                get_process=lambda v: v, set_process=lambda v: v,
-                check_set_errors=False, check_get_errors=False,
+    def control(get_command,
```

Comment:
Linter: I think we can accept that this method is unusually complex, so let's ignore that report.
```suggestion
    def control(get_command,  # noqa: C901 accept that this is a complex method
```